templates/admin/src/app.raw.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
```

Comment:
I would suggest to use one template for JSX and TSX similar to index.tsx_jsx.ts. We won't be able to use `.raw` anymore, but that's not that bad.